horovod/spark/common/params.py

```diff
@@ -199,6 +206,12 @@ def setNumProc(self, value):
     def getNumProc(self):
         return self.getOrDefault(self.num_proc)
 
+    def setDataModule(self, value):
+        return self._set(data_module=value)
+
+    def getDataModule(self):
+        return self.getOrDefault(self.data_module)
+
```

Comment:
remove `setDataModule` and `getDataModule` here since `datamodule` is specific to KerasEstimator and LightningEstimator.